tools/directord-dev-bootstrap-messaging-catalog.yaml

```diff
@@ -38,5 +44,16 @@ directord_clients:
           f.write(yaml.safe_dump(config, default_flow_style=False))
       EOC
   - RUN: sudo systemctl daemon-reload
+  - RUN: sudo systemctl enable certmonger.service
```

Comment:
can we get a synopsis of these steps added to the documentation for the messaging driver?

https://github.com/Directord/directord/blob/main/docs/drivers.md#messaging

Reply:
yes, I'll add it in this PR